Sources/InjectAutoLoader/InjectAutoLoader.m

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
can we remove copyright entry? I don't have one even on original lib files 😂

Reply:
Sure, I forgot about that :D I used to keep that header